packages/amplify-graphql-index-transformer/src/__tests__/__snapshots__/amplify-graphql-index-transformer.test.ts.snap

```diff
@@ -4583,7 +4583,7 @@ $util.qr($SkMap.put(\\"byGenre\\", []))
 ## [Start] Set query expression for @key **
 #set( $filterArgsMap = $ctx.args.filter.get(\\"and\\") )
 #set( $isLastSyncInDeltaTTLWindow = false )
-#set( $minLastSync = $util.time.nowEpochMilliSeconds() - 900000 )
+#set( $minLastSync = $util.time.nowEpochMilliSeconds() - 1800000 )
```

Comment:
Is this value changing because we're no longer applying the TTL override in the test? Shouldn't this carry forward?

Reply:
Hmm, probably missing a mock in the test. I'll take a look.